### ISSUE TITLE
Round robin shared subscription strategy not working

### ISSUE THREAD

**Issue by dario-collavini** [emqx/emqx#3757]
**Environment**

Docker image downloaded from https://hub.docker.com/r/emqx/emqx: **emqx:v4.1.3** 
Setup is the one provided by default, only the following env variables were added when running the container:
EMQX_MQTT__MAX_PACKET_SIZE: 20480KB
EMQX_LOADED_PLUGINS: emqx_recon,emqx_retainer,emqx_dashboard,emqx_auth_mysql
EMQX_BROKER__SHARED_SUBSCRIPTION_STRATEGY: round_robin

**What I was trying to do**

I tried to use shared subscriptions with "round_robin" policy instead of the default "random".
I connected two clients, subscribed them with "$share/group" sub and sent some mock MQTT messages.

**Issue**

When using round_robin, only one client is receiving all messages, while the other one doesn't receive any of them. 
When using random, everything works as expected (both clients randomly receive messages).

**Comment by tigercl** [comment]
@dario-collavini Sorry for not responding in time because of vacation. I just checked it again and everything is working fine. Which MQTT client software or client library are you using? Maybe you can use the trace command to trace these two clients. It will print all the related logs of the client to a specified file. You can refer to the following documents: https://docs.emqx.io/broker/latest/en/advanced/cli.html.

**Comment by dario-collavini** [comment]
Hello @tigercl thank you for your reply. 
I'm using the paho-mqtt library (v1.5.0) for Python 3.8.
I'm going to trying what you suggest to see if I can discover more.

**Comment by dario-collavini** [comment]
I made another test with the following setup:

**emqx:v4.1.3**:
As for my first message (docker container). The container is started via a docker-compose.yml with the following "environment" part:
```
environment:
      EMQX_MQTT__MAX_PACKET_SIZE: 20480KB
      EMQX_LOADED_PLUGINS: emqx_recon,emqx_retainer,emqx_dashboard,emqx_auth_mysql
      EMQX_BROKER__SHARED_SUBSCRIPTION_STRATEGY: round_robin
```
On the running container, I ran a "docker inspect" command to ensure that policy _round_robin_ had been set up correctly, and so it was.


**2 client instances of a custom program running Python3.8 and using paho-mqtt v1.5.0**:
These clients are running my custom code wrapping the paho library for mqtt communication.
These are their mqtt settings:
```
[MQTT]
BROKER_IP=172.22.27.22
BROKER_PORT=1883
CLIENT_ID=auth1
CLIENT_USER=<client_user>
CLIENT_PASSWD=<client_password>
CLEAN_SESSION=True
SHARED_TOPIC_PREFIX = "$share/SvcAuthentication"
```
Basically, it's the same for both clients except for the CLIENT_ID which is auth1 and auth2, respectively.
I checked that SHARED_TOPIC_PREFIX is correctly parsed as I took a look at the EMQX dashboard and for both the clients, their subscription is:

_$share/SvcAuthentication/+/Services/Requests/SLMS_AUTH_SERVICE_

![auth1](https://user-images.githubusercontent.com/25481699/95746500-2aa54000-0c97-11eb-8bd5-c2453fe8296e.png)
![auth2](https://user-images.githubusercontent.com/25481699/95746513-30028a80-0c97-11eb-83b6-6dde78a4f1b7.png)

---
Test:
Send an MQTT message to the following topic:

_pdxeng/Services/Requests/SLMS_AUTH_SERVICE_

will trigger the client to send the reply on the topic:

_pdxeng/Services/Replies/dario_

Since round robin is used, I expect that sending two requests will first trigger a client to handle the request, and then the second one. 
Looking at the trace log generated by the related emqx cli command, it seems that only one of the client is receiving both the requests:

**/opt/emqx/log $ cat auth1_mqtt.log** 
**2020-10-12 11:56:24.887 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] SEND PUBLISH**(Q0, R0, D0, Topic=pdxeng/Services/Requests/SLMS_AUTH_SERVICE, PacketId=undefined, Payload=<<"{\"message_id\": \"1\", \"command\": \"PostToken\", \"params\": {\"username\": \"<user>\", \"password\": \"<password>\"}, \"requestor\": \"dario\"}">>)
**2020-10-12 11:56:26.283 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] RECV** <<48,209,5,0,29,112,100,120,101,110,103,47,83,101,114,118,105,99,101,115,47,82,101,112,108,105,101,115,47,100,97,114,105,111,120,156,141,148,81,115,162,48,16,199,191,74,135,231,210,1,161,20,239,13,16,106,41,40,42,160,114,211,97,144,68,136,64,192,4,80,232,248,221,15,188,233,221,220,203,77,95,50,59,251,219,127,54,217,221,228,147,41,32,165,81,2,67,4,152,31,15,12,207,60,62,48,4,210,170,196,0,146,209,179,177,236,77,168,120,238,60,220,232,107,255,77,211,67,254,43,168,201,235,33,226,147,1,81,29,221,141,186,204,32,30,69,176,51,211,195,107,140,150,200,84,237,147,55,217,240,49,178,52,51,63,20,246,31,223,138,51,103,150,151,186,11,87,239,222,48,247,164,84,174,51,243,182,49,61,179,102,170,201,129,115,28,86,218,131,93,113,157,248,11,25,114,189,158,26,185,134,227,112,197,207,250,184,158,214,64,68,250,244,61,74,88,79,73,104,172,246,222,85,154,58,188,66,79,187,56,199,156,191,240,73,87,247,173,101,184,96,151,44,139,39,129,207,182,160,45,149,165,103,29,196,61,20,150,207,167,74,79,158,218,44,95,93,151,27,147,221,110,131,169,209,85,200,132,135,235,179,4,22,190,120,85,167,196,222,5,89,114,126,22,83,56,181,124,160,137,187,89,36,207,83,233,212,100,17,49,106,59,80,231,100,197,138,153,171,227,218,232,14,221,69,61,103,210,26,17,101,178,91,90,32,227,82,64,216,169,136,160,161,102,124,185,191,216,171,217,17,11,169,88,0,28,59,103,84,238,79,102,155,29,155,237,186,34,142,6,170,78,124,239,222,92,203,177,180,5,138,69,187,151,85,73,80,227,151,117,159,202,19,57,240,118,54,123,200,101,123,253,218,244,251,224,236,85,122,56,23,201,1,180,88,242,46,98,200,89,194,121,143,197,235,123,121,142,228,224,178,12,242,156,165,188,25,251,184,149,130,80,15,5,161,63,170,186,3,19,187,113,243,222,129,203,115,203,150,161,113,1,216,127,237,210,153,225,238,121,111,88,26,191,237,95,148,39,71,99,217,166,93,121,153,171,29,4,99,19,24,147,25,124,157,103,54,111,139,220,186,232,26,196,73,145,232,139,156,234,203,227,56,192,107,133,8,28,123,47,72,28,55,122,162,56,30,70,43,36,40,73,107,58,128,159,252,227,195,207,79,134,194,184,70,229,125,74,0,108,81,12,181,18,215,164,204,149,170,26,85,199,6,223,249,93,193,40,121,206,124,220,30,31,254,145,33,220,66,92,151,164,251,190,228,24,161,188,33,80,193,81,222,81,68,191,47,196,176,190,148,36,179,75,140,134,140,223,215,17,88,149,164,254,126,124,81,142,214,6,98,90,146,255,29,239,227,99,0,180,169,32,105,232,253,125,14,69,101,226,130,134,205,239,247,91,129,43,196,201,223,134,132,53,42,198,174,240,18,55,121,230,94,4,89,26,16,162,97,62,20,30,211,1,19,20,229,3,63,70,57,133,3,42,26,90,135,113,26,225,225,71,168,34,74,135,203,131,47,124,27,83,149,96,220,142,187,221,126,1,218,240,100,231>>
**2020-10-12 11:56:26.284 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] RECV PUBLISH**(Q0, R0, D0, Topic=pdxeng/Services/Replies/dario, PacketId=undefined, Payload=<<120,156,141,148,81,115,162,48,16,199,191,74,135,231,210,1,161,20,239,13,16,106,41,40,42,160,114,211,97,144,68,136,64,192,4,80,232,248,221,15,188,233,221,220,203,77,95,50,59,251,219,127,54,217,221,228,147,41,32,165,81,2,67,4,152,31,15,12,207,60,62,48,4,210,170,196,0,146,209,179,177,236,77,168,120,238,60,220,232,107,255,77,211,67,254,43,168,201,235,33,226,147,1,81,29,221,141,186,204,32,30,69,176,51,211,195,107,140,150,200,84,237,147,55,217,240,49,178,52,51,63,20,246,31,223,138,51,103,150,151,186,11,87,239,222,48,247,164,84,174,51,243,182,49,61,179,102,170,201,129,115,28,86,218,131,93,113,157,248,11,25,114,189,158,26,185,134,227,112,197,207,250,184,158,214,64,68,250,244,61,74,88,79,73,104,172,246,222,85,154,58,188,66,79,187,56,199,156,191,240,73,87,247,173,101,184,96,151,44,139,39,129,207,182,160,45,149,165,103,29,196,61,20,150,207,167,74,79,158,218,44,95,93,151,27,147,221,110,131,169,209,85,200,132,135,235,179,4,22,190,120,85,167,196,222,5,89,114,126,22,83,56,181,124,160,137,187,89,36,207,83,233,212,100,17,49,106,59,80,231,100,197,138,153,171,227,218,232,14,221,69,61,103,210,26,17,101,178,91,90,32,227,82,64,216,169,136,160,161,102,124,185,191,216,171,217,17,11,169,88,0,28,59,103,84,238,79,102,155,29,155,237,186,34,142,6,170,78,124,239,222,92,203,177,180,5,138,69,187,151,85,73,80,227,151,117,159,202,19,57,240,118,54,123,200,101,123,253,218,244,251,224,236,85,122,56,23,201,1,180,88,242,46,98,200,89,194,121,143,197,235,123,121,142,228,224,178,12,242,156,165,188,25,251,184,149,130,80,15,5,161,63,170,186,3,19,187,113,243,222,129,203,115,203,150,161,113,1,216,127,237,210,153,225,238,121,111,88,26,191,237,95,148,39,71,99,217,166,93,121,153,171,29,4,99,19,24,147,25,124,157,103,54,111,139,220,186,232,26,196,73,145,232,139,156,234,203,227,56,192,107,133,8,28,123,47,72,28,55,122,162,56,30,70,43,36,40,73,107,58,128,159,252,227,195,207,79,134,194,184,70,229,125,74,0,108,81,12,181,18,215,164,204,149,170,26,85,199,6,223,249,93,193,40,121,206,124,220,30,31,254,145,33,220,66,92,151,164,251,190,228,24,161,188,33,80,193,81,222,81,68,191,47,196,176,190,148,36,179,75,140,134,140,223,215,17,88,149,164,254,126,124,81,142,214,6,98,90,146,255,29,239,227,99,0,180,169,32,105,232,253,125,14,69,101,226,130,134,205,239,247,91,129,43,196,201,223,134,132,53,42,198,174,240,18,55,121,230,94,4,89,26,16,162,97,62,20,30,211,1,19,20,229,3,63,70,57,133,3,42,26,90,135,113,26,225,225,71,168,34,74,135,203,131,47,124,27,83,149,96,220,142,187,221,126,1,218,240,100,231>>)
**2020-10-12 11:56:26.287 [info] <<"auth1">>@172.22.163.107:61929 PUBLISH to pdxeng/Services/Replies/dario:** <<120,156,141,148,81,115,162,48,16,199,191,74,135,231,210,1,161,20,239,13,16,106,41,40,42,160,114,211,97,144,68,136,64,192,4,80,232,248,221,15,188,233,221,220,203,77,95,50,59,251,219,127,54,217,221,228,147,41,32,165,81,2,67,4,152,31,15,12,207,60,62,48,4,210,170,196,0,146,209,179,177,236,77,168,120,238,60,220,232,107,255,77,211,67,254,43,168,201,235,33,226,147,1,81,29,221,141,186,204,32,30,69,176,51,211,195,107,140,150,200,84,237,147,55,217,240,49,178,52,51,63,20,246,31,223,138,51,103,150,151,186,11,87,239,222,48,247,164,84,174,51,243,182,49,61,179,102,170,201,129,115,28,86,218,131,93,113,157,248,11,25,114,189,158,26,185,134,227,112,197,207,250,184,158,214,64,68,250,244,61,74,88,79,73,104,172,246,222,85,154,58,188,66,79,187,56,199,156,191,240,73,87,247,173,101,184,96,151,44,139,39,129,207,182,160,45,149,165,103,29,196,61,20,150,207,167,74,79,158,218,44,95,93,151,27,147,221,110,131,169,209,85,200,132,135,235,179,4,22,190,120,85,167,196,222,5,89,114,126,22,83,56,181,124,160,137,187,89,36,207,83,233,212,100,17,49,106,59,80,231,100,197,138,153,171,227,218,232,14,221,69,61,103,210,26,17,101,178,91,90,32,227,82,64,216,169,136,160,161,102,124,185,191,216,171,217,17,11,169,88,0,28,59,103,84,238,79,102,155,29,155,237,186,34,142,6,170,78,124,239,222,92,203,177,180,5,138,69,187,151,85,73,80,227,151,117,159,202,19,57,240,118,54,123,200,101,123,253,218,244,251,224,236,85,122,56,23,201,1,180,88,242,46,98,200,89,194,121,143,197,235,123,121,142,228,224,178,12,242,156,165,188,25,251,184,149,130,80,15,5,161,63,170,186,3,19,187,113,243,222,129,203,115,203,150,161,113,1,216,127,237,210,153,225,238,121,111,88,26,191,237,95,148,39,71,99,217,166,93,121,153,171,29,4,99,19,24,147,25,124,157,103,54,111,139,220,186,232,26,196,73,145,232,139,156,234,203,227,56,192,107,133,8,28,123,47,72,28,55,122,162,56,30,70,43,36,40,73,107,58,128,159,252,227,195,207,79,134,194,184,70,229,125,74,0,108,81,12,181,18,215,164,204,149,170,26,85,199,6,223,249,93,193,40,121,206,124,220,30,31,254,145,33,220,66,92,151,164,251,190,228,24,161,188,33,80,193,81,222,81,68,191,47,196,176,190,148,36,179,75,140,134,140,223,215,17,88,149,164,254,126,124,81,142,214,6,98,90,146,255,29,239,227,99,0,180,169,32,105,232,253,125,14,69,101,226,130,134,205,239,247,91,129,43,196,201,223,134,132,53,42,198,174,240,18,55,121,230,94,4,89,26,16,162,97,62,20,30,211,1,19,20,229,3,63,70,57,133,3,42,26,90,135,113,26,225,225,71,168,34,74,135,203,131,47,124,27,83,149,96,220,142,187,221,126,1,218,240,100,231>>
**2020-10-12 11:56:37.261 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] SEND PUBLISH**(Q0, R0, D0, Topic=pdxeng/Services/Requests/SLMS_AUTH_SERVICE, PacketId=undefined, Payload=<<"{\"message_id\": \"1\", \"command\": \"PostToken\", \"params\": {\"username\": \"<user>\", \"password\": \"<password>\"}, \"requestor\": \"dario\"}">>)
**2020-10-12 11:56:38.603 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] RECV** <<48,213,5,0,29,112,100,120,101,110,103,47,83,101,114,118,105,99,101,115,47,82,101,112,108,105,101,115,47,100,97,114,105,111,120,156,141,148,89,111,171,58,16,199,191,74,197,115,169,32,9,16,206,27,161,4,194,158,5,178,28,85,136,128,217,2,54,177,129,144,84,253,238,7,114,212,123,117,95,174,250,54,154,223,252,61,246,44,254,164,42,64,72,152,130,32,143,169,95,47,20,75,189,190,80,24,144,26,193,24,224,209,179,53,173,109,32,121,59,45,216,42,27,127,37,43,1,251,29,212,150,205,16,241,73,197,97,19,62,141,6,93,0,28,69,224,174,103,103,53,202,157,92,95,88,133,55,217,178,81,110,202,122,121,174,172,127,124,107,70,127,55,189,108,103,239,148,251,10,50,111,234,60,225,69,102,150,230,165,33,57,174,79,72,212,224,100,21,28,32,191,140,61,246,184,182,253,179,24,58,135,93,29,158,37,94,182,188,46,112,124,160,161,125,117,46,212,253,105,205,44,100,58,149,81,183,60,58,215,165,51,61,72,74,52,65,153,131,230,133,211,79,247,180,57,125,227,150,126,173,170,157,206,92,181,247,115,219,217,1,15,236,153,244,198,115,71,28,103,74,98,52,92,161,210,250,142,67,197,237,190,86,44,165,136,185,246,98,123,169,31,204,110,229,109,157,218,226,22,175,180,179,99,91,197,141,57,45,149,165,0,233,155,136,142,174,106,94,234,13,167,197,142,81,116,91,249,118,53,14,83,75,123,204,247,154,237,231,151,110,37,94,224,233,225,138,143,141,243,126,33,123,225,221,20,207,109,235,26,68,122,4,252,202,13,59,133,133,236,220,208,143,134,235,35,21,135,51,43,211,232,222,234,252,158,83,97,223,41,1,136,233,135,38,108,66,238,58,41,141,181,217,100,167,199,124,213,119,233,166,52,123,89,223,24,34,146,122,128,35,207,76,146,125,191,146,74,44,116,74,92,75,44,18,69,176,19,30,206,182,153,117,128,174,38,206,226,209,58,243,27,178,186,232,186,88,110,238,241,140,244,50,239,34,37,208,156,195,66,91,122,217,68,222,36,247,123,98,159,174,235,183,176,13,211,211,33,208,135,230,67,183,234,175,89,45,175,105,127,18,225,57,161,189,24,184,23,92,64,161,17,4,163,182,198,113,0,125,157,99,48,246,126,202,51,204,232,9,163,104,24,173,0,231,105,214,144,1,252,102,95,95,126,127,82,4,68,77,142,158,83,18,131,46,143,128,140,96,131,81,41,213,245,168,74,90,248,228,79,5,37,149,37,245,241,245,250,242,31,89,14,59,0,27,132,239,63,151,36,97,94,182,24,72,48,44,239,36,39,63,23,66,208,220,16,190,88,8,230,67,198,159,235,48,168,17,110,126,30,95,161,209,218,2,72,16,254,191,235,125,124,12,128,180,53,192,45,121,238,231,80,84,42,170,72,208,254,221,223,58,238,1,76,255,109,72,208,228,213,216,21,150,103,38,28,35,76,197,249,128,114,18,148,67,225,33,25,48,206,195,114,224,73,88,18,48,160,170,37,77,16,101,33,28,126,132,58,36,100,120,124,252,141,191,198,84,40,30,143,99,190,190,254,0,247,83,100,224>>
**2020-10-12 11:56:38.604 [debug] <<"auth1">>@172.22.163.107:61929 [MQTT] RECV PUBLISH**(Q0, R0, D0, Topic=pdxeng/Services/Replies/dario, PacketId=undefined, Payload=<<120,156,141,148,89,111,171,58,16,199,191,74,197,115,169,32,9,16,206,27,161,4,194,158,5,178,28,85,136,128,217,2,54,177,129,144,84,253,238,7,114,212,123,117,95,174,250,54,154,223,252,61,246,44,254,164,42,64,72,152,130,32,143,169,95,47,20,75,189,190,80,24,144,26,193,24,224,209,179,53,173,109,32,121,59,45,216,42,27,127,37,43,1,251,29,212,150,205,16,241,73,197,97,19,62,141,6,93,0,28,69,224,174,103,103,53,202,157,92,95,88,133,55,217,178,81,110,202,122,121,174,172,127,124,107,70,127,55,189,108,103,239,148,251,10,50,111,234,60,225,69,102,150,230,165,33,57,174,79,72,212,224,100,21,28,32,191,140,61,246,184,182,253,179,24,58,135,93,29,158,37,94,182,188,46,112,124,160,161,125,117,46,212,253,105,205,44,100,58,149,81,183,60,58,215,165,51,61,72,74,52,65,153,131,230,133,211,79,247,180,57,125,227,150,126,173,170,157,206,92,181,247,115,219,217,1,15,236,153,244,198,115,71,28,103,74,98,52,92,161,210,250,142,67,197,237,190,86,44,165,136,185,246,98,123,169,31,204,110,229,109,157,218,226,22,175,180,179,99,91,197,141,57,45,149,165,0,233,155,136,142,174,106,94,234,13,167,197,142,81,116,91,249,118,53,14,83,75,123,204,247,154,237,231,151,110,37,94,224,233,225,138,143,141,243,126,33,123,225,221,20,207,109,235,26,68,122,4,252,202,13,59,133,133,236,220,208,143,134,235,35,21,135,51,43,211,232,222,234,252,158,83,97,223,41,1,136,233,135,38,108,66,238,58,41,141,181,217,100,167,199,124,213,119,233,166,52,123,89,223,24,34,146,122,128,35,207,76,146,125,191,146,74,44,116,74,92,75,44,18,69,176,19,30,206,182,153,117,128,174,38,206,226,209,58,243,27,178,186,232,186,88,110,238,241,140,244,50,239,34,37,208,156,195,66,91,122,217,68,222,36,247,123,98,159,174,235,183,176,13,211,211,33,208,135,230,67,183,234,175,89,45,175,105,127,18,225,57,161,189,24,184,23,92,64,161,17,4,163,182,198,113,0,125,157,99,48,246,126,202,51,204,232,9,163,104,24,173,0,231,105,214,144,1,252,102,95,95,126,127,82,4,68,77,142,158,83,18,131,46,143,128,140,96,131,81,41,213,245,168,74,90,248,228,79,5,37,149,37,245,241,245,250,242,31,89,14,59,0,27,132,239,63,151,36,97,94,182,24,72,48,44,239,36,39,63,23,66,208,220,16,190,88,8,230,67,198,159,235,48,168,17,110,126,30,95,161,209,218,2,72,16,254,191,235,125,124,12,128,180,53,192,45,121,238,231,80,84,42,170,72,208,254,221,223,58,238,1,76,255,109,72,208,228,213,216,21,150,103,38,28,35,76,197,249,128,114,18,148,67,225,33,25,48,206,195,114,224,73,88,18,48,160,170,37,77,16,101,33,28,126,132,58,36,100,120,124,252,141,191,198,84,40,30,143,99,190,190,254,0,247,83,100,224>>)
**2020-10-12 11:56:38.605 [info] <<"auth1">>@172.22.163.107:61929 PUBLISH to pdxeng/Services/Replies/dario**: <<120,156,141,148,89,111,171,58,16,199,191,74,197,115,169,32,9,16,206,27,161,4,194,158,5,178,28,85,136,128,217,2,54,177,129,144,84,253,238,7,114,212,123,117,95,174,250,54,154,223,252,61,246,44,254,164,42,64,72,152,130,32,143,169,95,47,20,75,189,190,80,24,144,26,193,24,224,209,179,53,173,109,32,121,59,45,216,42,27,127,37,43,1,251,29,212,150,205,16,241,73,197,97,19,62,141,6,93,0,28,69,224,174,103,103,53,202,157,92,95,88,133,55,217,178,81,110,202,122,121,174,172,127,124,107,70,127,55,189,108,103,239,148,251,10,50,111,234,60,225,69,102,150,230,165,33,57,174,79,72,212,224,100,21,28,32,191,140,61,246,184,182,253,179,24,58,135,93,29,158,37,94,182,188,46,112,124,160,161,125,117,46,212,253,105,205,44,100,58,149,81,183,60,58,215,165,51,61,72,74,52,65,153,131,230,133,211,79,247,180,57,125,227,150,126,173,170,157,206,92,181,247,115,219,217,1,15,236,153,244,198,115,71,28,103,74,98,52,92,161,210,250,142,67,197,237,190,86,44,165,136,185,246,98,123,169,31,204,110,229,109,157,218,226,22,175,180,179,99,91,197,141,57,45,149,165,0,233,155,136,142,174,106,94,234,13,167,197,142,81,116,91,249,118,53,14,83,75,123,204,247,154,237,231,151,110,37,94,224,233,225,138,143,141,243,126,33,123,225,221,20,207,109,235,26,68,122,4,252,202,13,59,133,133,236,220,208,143,134,235,35,21,135,51,43,211,232,222,234,252,158,83,97,223,41,1,136,233,135,38,108,66,238,58,41,141,181,217,100,167,199,124,213,119,233,166,52,123,89,223,24,34,146,122,128,35,207,76,146,125,191,146,74,44,116,74,92,75,44,18,69,176,19,30,206,182,153,117,128,174,38,206,226,209,58,243,27,178,186,232,186,88,110,238,241,140,244,50,239,34,37,208,156,195,66,91,122,217,68,222,36,247,123,98,159,174,235,183,176,13,211,211,33,208,135,230,67,183,234,175,89,45,175,105,127,18,225,57,161,189,24,184,23,92,64,161,17,4,163,182,198,113,0,125,157,99,48,246,126,202,51,204,232,9,163,104,24,173,0,231,105,214,144,1,252,102,95,95,126,127,82,4,68,77,142,158,83,18,131,46,143,128,140,96,131,81,41,213,245,168,74,90,248,228,79,5,37,149,37,245,241,245,250,242,31,89,14,59,0,27,132,239,63,151,36,97,94,182,24,72,48,44,239,36,39,63,23,66,208,220,16,190,88,8,230,67,198,159,235,48,168,17,110,126,30,95,161,209,218,2,72,16,254,191,235,125,124,12,128,180,53,192,45,121,238,231,80,84,42,170,72,208,254,221,223,58,238,1,76,255,109,72,208,228,213,216,21,150,103,38,28,35,76,197,249,128,114,18,148,67,225,33,25,48,206,195,114,224,73,88,18,48,160,170,37,77,16,101,33,28,126,132,58,36,100,120,124,252,141,191,198,84,40,30,143,99,190,190,254,0,247,83,100,224>>

**/opt/emqx/log $ cat auth2_mqtt.log** 
2020-10-12 11:56:27.304 [debug] <<"auth2">>@172.22.163.107:61976 [MQTT] RECV <<192,0>>
2020-10-12 11:56:27.305 [debug] <<"auth2">>@172.22.163.107:61976 [MQTT] SEND PINGRESP(Q0, R0, D0)
2020-10-12 11:56:57.440 [debug] <<"auth2">>@172.22.163.107:61976 [MQTT] RECV <<192,0>>
2020-10-12 11:56:57.440 [debug] <<"auth2">>@172.22.163.107:61976 [MQTT] SEND PINGRESP(Q0, R0, D0)
